### PR TITLE
example for policy-integrations

### PR DESCRIPTION
**Pull request by atoonk**
v1

### CODE REVIEW

File: policy-integrations/app.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    data = request.json
+    
+    ip = data['ip']
+    tz_info = get_timezone(ip)
+
+    # Get the location of the user based on the IP address
+    location = tz_info['city'] + ',' + tz_info['countryCode']
+
+    # WeatherAPI Key
+    weather_api_key = '3fd855a8e2a347ff863153348233005'
```
Comment (adrianosela) [inline policy-integrations/app.py:96]: if you're paying for this weather service, might wanna revoke this!

File: policy-integrations/app.py [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+    # WeatherAPI endpoint
+    weather_url = f"http://api.weatherapi.com/v1/current.json?key={weather_api_key}&q={location}"
```
Comment (adrianosela) [inline policy-integrations/app.py:99]: `s/http/https/g`

File: policy-integrations/app.py [hunk 3]
```diff
@@ -0,0 +1,194 @@
+from flask import Flask, request, jsonify, abort, make_response
```
Comment (adrianosela) [inline policy-integrations/app.py:1]: remove commented out code, and use constants for `ip`, `protocol`, `email`, etc...

File: policy-integrations/README.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+- `/random`: This endpoint randomly generates a confidence score between 60 and 100. If the incoming request protocol is "http", it denies the request.
+
+- `/rainorshine`: This endpoint uses the requester's IP address to determine their current location and fetches the current weather data. If it's raining, the response will indicate so.
+
+- `/businesshours`: This endpoint checks if the request is made within business hours (between 9am and 6pm local time, Monday through Friday).
+
+Each endpoint requires a valid 'Authorization' header with a secret key. The default secret key is 'superSecret', but can be changed by setting the 'API_SECRET' environment variable.
+
+## Setup
```
Comment (adrianosela) [inline policy-integrations/README.md:19]: we should have a sample body for folks to paste in the body of their integration in this README. e.g.

```
{
   "ip":"${inject::ip}",
   "email":"${inject::email}",
   "protocol":"${inject::protocol}",
   "socket_id":"${inject::socket_id}",
   "socket_name":"${inject::socket_name}"
}
```

**Review by adrianosela — APPROVED** [review]
great place to put this!

File: policy-integrations/app.py (hunk 1 above)
Comment (atoonk) [reply policy-integrations/app.py:96]: duh done. Free token

File: policy-integrations/app.py (hunk 2 above)
Comment (atoonk) [reply policy-integrations/app.py:99]: done

File: policy-integrations/README.md (hunk 4 above)
Comment (atoonk) [reply policy-integrations/README.md:19]: I'll add a link to the blog and docs.

**Pull request merged**